packages/mobile/src/screens/upload-screen/ParamList.ts

```diff
@@ -0,0 +1,6 @@
+import type { DocumentPickerResponse } from 'react-native-document-picker'
+
+export type UploadParamList = {
+  Upload: undefined
```

Comment:
should this always be undefined or an optional type?

Reply:
yeah we have undefined when there isn't any params, not sure how else to do it...